[OSDOCS#17175]: Make changes as per the errors and warnings for CQA-2 (#114273)
openshift/openshift-docs: security/compliance_operator/co-scans/compliance-operator-tailor.adoc

toc::[]

[role="_abstract"]

> While the Compliance Operator includes ready-to-use profiles, they must be modified to fit the organization needs and requirements. The process of modifying a profile is called _tailoring_.

Reviewer: ```suggestion
Although the Compliance Operator includes ready-to-use profiles, you must modify the profiles to fit your organization's requirements. The process of modifying a profile is called _tailoring_.
```
